app/angular/components/deleteModelModal.html

```diff
@@ -1,12 +1,12 @@
 <div class="modal-header">
-	<h3 class="modal-title">Deletar modelagem</h3>
+	<h3 class="modal-title">{{ 'Delete model' | translate }}</h3>
 </div>
 
 <div class="modal-body">
-	<p>Tem certeza que deseja <strong>excluir</strong> a modelagem?</p>
+	<p>{{ 'Are you sure you want to delete the model?' | translate }}</p>
```

Comment:
We won't have an emphasis on the delete word anymore?

Reply:
For the translations to work including html elements we have to include some configs, I'll do it later.